tests/ChatTests/Local.hs

```diff
@@ -0,0 +1,92 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PostfixOperators #-}
+
+module ChatTests.Local where
+
+import ChatClient
+import ChatTests.Utils
+import Test.Hspec
+import System.Directory (copyFile)
+
+chatLocalTests :: SpecWith FilePath
+chatLocalTests = do
+  fdescribe "note folders" $ do
+    it "create folders, add notes, read, search" testNotes
```

Comment:
tests are too sparse, add tests for following functionality:
- messages with files (both text and file), should get chat and check list of messages via chatF
- create several (at least 2) folders, create messages in both, get chat and check list of messages (currently a single test creates two folders and doesn't create messages in 2nd)
- pagination inside notes chat - not only last, before and after should test for different scenarios (e.g. after but not till end, before and not till beginning)
- pagination of chat list with notes included (e.g. in testPaginationAllChatTypes)

possibly other scenarios

Reply:
made edits according to recent discussion to get rid of names and api to create